src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Query/PropertyPages/UIPropertyEditorDataProducer.cs

```diff
@@ -64,6 +64,18 @@ IEnumerable<IEntityValue> createEditorValues()
                     IEntityValue editorValue = CreateEditorValue(parent, editor, properties);
                     yield return editorValue;
                 }
+
+                if (property is StringProperty stringProperty)
+                {
+                    if (stringProperty.Subtype == "file")
+                    {
+                        yield return CreateEditorValue(parent, new ValueEditor { EditorType = "FileBrowse" }, properties);
+                    }
+                    else if (stringProperty.Subtype == "path")
+                    {
+                        yield return CreateEditorValue(parent, new ValueEditor { EditorType = "PathBrowse" }, properties);
```

Comment:
How about "DirectoryBrowser"? A path could be either a file or a directory.

Reply:
The API docs on `Subtype` suggested "path", but //index suggests no one is actually using that, so I'll add support for `Subtype="directory"` instead and use "Directory" or "DirectoryBrowse" for the editor type.